src/app/ui/editor/select_box_state.cpp

```diff
@@ -65,6 +85,30 @@ void SelectBoxState::setBoxBounds(const gfx::Rect& box)
   m_rulers[V2] = Ruler(Ruler::Vertical, box.x+box.w);
 }
 
+// Get and Set Padding for Import Sprite Sheet box state
+gfx::Size SelectBoxState::getPaddingBounds() const
+{
+  ASSERT(hasFlag(Flags::PaddingRulers));
+  int w = m_rulers[PV].position() - m_rulers[V2].position();
+  int h = m_rulers[PH].position() - m_rulers[H2].position();
+  if (w < 0)
+    w = 0;
+  if (h < 0)
+    h = 0;
+  return gfx::Size(w, h);
+}
+
+void SelectBoxState::setPaddingBounds(gfx::Size& padding)
```

Comment:
I don't like 100% that this function receives the padding as an input/output parameter (because the `padding` argument will be modified). In the best case the `padding` should be `const gfx::Size&` and the `< 0` check should be in `ImportSpriteSheetWindow::getPaddingFromEntries()`.
```suggestion
void SelectBoxState::setPaddingBounds(const gfx::Size& padding)
```